### PR TITLE
Fix git repo ssh

### PR DESCRIPTION
**Pull request by pdietl**

### CODE REVIEW

**Review by tcely — APPROVED** [review]

**Comment by pdietl** [comment]
ping @tcely @ncopa 

Is this good to go? :)

**Comment by tcely** [comment]
:ping_pong: @ncopa

File: functions.sh.in [hunk 1]
```diff
@@ -88,7 +88,8 @@ readconfig() {
 	else
 		if [ -n "$gitbase" ]; then
 			case $(git config remote.origin.url) in
-			*/aports) APORTSDIR=$gitbase ;;
+			# '.git' for SSH URLs, and no suffix for HTTPS URLs
+			*/aports | */aports.git) APORTSDIR=$gitbase ;;
```
Comment (tcely) [inline functions.sh.in]:
```suggestion
			*/aports|*/aports.git) APORTSDIR=$gitbase ;;
```

I'm not sure if the spaces are going to affect the match or not, but better to just not have them.

File: functions.sh.in [hunk 2]
```diff
@@ -88,7 +88,8 @@ readconfig() {
 	else
 		if [ -n "$gitbase" ]; then
 			case $(git config remote.origin.url) in
```
Comment (tcely) [inline functions.sh.in]:
```suggestion
			case $(git remote get-url origin) in
```
The value from config is not necessarily a URL, but gets translated to one by `url.*` configuration.

**Review by tcely — CHANGES_REQUESTED** [review]
- [x] use git remote
- [x] remove spaces in case matching

**Review by tcely — APPROVED** [review]

**Comment by tcely** [comment]
You can rebase this and squash to a single commit to make it a bit cleaner.

**Comment by pdietl** [comment]
Ok @tcely && @ncopa, I have rebased and squashed into one commit. Sorry for the delay. :p

**Comment by pdietl** [comment]
@tcely @ncopa. Can we merge? :)

**Pull request closed without merging**